### ISSUE TITLE
I am unable to get Atanua to start and keep getting this error

### ISSUE THREAD

**Issue by tusharraval102** [Davidster/AtanuaInDocker#2]
![image](https://user-images.githubusercontent.com/58268852/107841132-3eb49780-6d86-11eb-8493-3f4dda3eafed.png)

I keep getting this error which is not allowing me to start Atanua. Originally, I updated XQuartz which I believe caused the issue, but after deleting and reinstalling the previous version, nothing was opening. Now, after updating XQuartz again, this is the error I receive. Please let me know if I can send any other logs.

**Comment by Davidster** [comment]
thanks for the report! what kind of mac / osx version are you using? I will take a look tonight to see if I can reproduce the error on my mac

**Comment by tusharraval102** [comment]
I am using the MacBook Pro 16", with Big Sur v11.2.1. Thanks!

**Comment by Davidster** [comment]
It seems like something really nasty happens after installing the new version of xquartz 2.8.0_beta3. I tried it myself and now my xquartz seems to be completely broken, even after uninstalling and reinstalling the old version multiple times and logging in / logging out in between. Other users reporting similar issues here: https://github.com/XQuartz/XQuartz/issues/66 and the package maintainer doesn't seem to be super helpful. Apparently re-installing the OS doesn't fix the problem 😨 

I'm going to add a note to make sure nobody else tries to upgrade. We'll have to monitor the issues on XQuartz to see if people can figure out how to fix this

**Comment by tusharraval102** [comment]
Yup! I ended up using a windows vm to get it to work. I hope no one upgrades and this gets solved soon. It’s honestly a crazy bug.

**Comment by Reuven1203** [comment]
Hey! I tried starting atanua on my m1 mac and it cannot start
